app/Core/KeepAliveDoctrineServiceProvider.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Core;
+
+use CultuurNet\ProjectAanvraag\Core\Doctrine\ConnectionKeepAlive;
+use Pimple\Container;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Configuration;
+use Doctrine\Common\EventManager;
+use Silex\Provider\DoctrineServiceProvider;
+
+/**
+ * Custom implementation of the Doctrine DBAL Provider.
+ */
+class KeepAliveDoctrineServiceProvider extends DoctrineServiceProvider
+{
+    public function register(Container $app)
+    {
+        $app['db.default_options'] = array(
```

Comment:
There is no need to do repeat all of this. You can just do parent::register, only the custom code should live here.